test/test_events.py

```diff
@@ -16,6 +16,20 @@ def test_receiving_events():
     eq(event[1], 'py!')
     eq(event[2], [vim.current.buffer.number])
 
+@with_setup(setup=cleanup)
+def test_sending_notify():
+    # notify after notify
+    vim.session.notify('vim_command', "let g:test = 3")
+    cmd = 'call rpcnotify(%d, "test-event", g:test)' % vim.channel_id
```

Comment:
Usually I prefer `"{}".format()` myself, but I wanted to be consistent with the surrounding code. If we want it to change, it is better to discuss that in separate PR.